### PR TITLE
feat: modified README

### PR DESCRIPTION
**Pull request by EugenioCorso**
First draft of the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
 # Please note
 This version works only with `python 3.6.x`
```
Comment (libremente) [inline README.md]: I would move this below, it is not that relevant to have it at the top

File: README.md [hunk 2]
```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
...
 
+- [Get Started](#get-started)
+- [How to contribute](#how-to-contribute)
+- [Maintenance](#Maintenance)
+- [License](#license)
 
+# Get Started
 
+This is the main codebase for FARE. FARE is an e-learning web platform. Inside FARE you can found a lot of educational material, download it or you can upload some.
```
Comment (libremente) [inline README.md]:
```suggestion
This is the main codebase for FARE. FARE is an e-learning web platform. Inside FARE you can find a lot of educational material, download it or you can upload some.
```

File: README.md [hunk 3]
```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
...
+## Upload content
+Everyone can upload content, you have to be logged in. After the upload your content will be reviewed and, if everything is ok, it will be published
+
+## Review the material
+If you are a staff member, you can review the material and let it be visible by everyone!
+
+# How to contribute
+
+This project runs Invenio vx.x (insert structure of the project).
+To make run our instance of Invenio follow the installation [here](https://invenio.readthedocs.io/en/latest/quickstart/quickstart.html)
```
Comment (libremente) [inline README.md]: I would state how to run *this* instance and not a Invenio instance in general

File: README.md [hunk 4]
```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
...
+
+## Community
+(insert contact of a possible comunity, like gitter or slack)
```
Comment (libremente) [inline README.md]: If none, let's remove this for the moment

File: README.md [hunk 5]
```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
...
+
+## Code of conduct
+[Code of conduct](https://www.contributor-covenant.org/)
+
+## Reporting a bug and asking help
+
+If you found a bug plese report it [here](https://github.com/open-education-polito/fare-invenio) opening an issue.
+Searching help? (contatti per chi chiede aiuto)
```
Comment (libremente) [inline README.md]: Let's point to the issue as well.

File: README.md [hunk 6]
```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
...
+## Semantic Versioning
+[Here](https://semver.org/) the semantic versioning we use
+
+# Maintenance
+
+Code actually maintained by (add contacts of maintainers)
+
+# License
+
+This code is licensed under a GNU Affero General Public License, v3. See the LICENSE.md file for extended info, and the license files inside each module's repository.
```
Comment (libremente) [inline README.md]: I am not sure we are allowed to do so, we need to double check this. @marioscovazzi

**Comment by libremente** [comment]
We need many more instructions on how to run the software and how to deal with the local deployment

**Comment by libremente** [comment]
Add info regarding ADR. 
This means that we need to set which is the location (`doc/architecture/decisions`), which is the tool to use (`adr-tools`) and the possible links to use it https://github.com/npryce/adr-tools

File: README.md (hunk 6 above)
Comment (libremente) [reply README.md]: MIT confirmed.

File: README.md (hunk 1 above)
Comment (EugenioCorso) [reply README.md]: I'll fix it

File: README.md (hunk 3 above)
Comment (EugenioCorso) [reply README.md]: I extended it, tell me if further improvement are needed

File: README.md (hunk 4 above)
Comment (EugenioCorso) [reply README.md]: ok

File: README.md (hunk 6 above)
Comment (EugenioCorso) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (EugenioCorso) [reply README.md]: done

**Comment by libremente** [comment]
@EugenioCorso @CorsoGabriele did you have time to check the ADR? Please insert it in the README and then let's merge this fast. Thanks!

**Comment by libremente** [comment]
@EugenioCorso can you turn this PR into ready for review?

**Pull request merged**